components/MaintainerCard/MaintainerCard.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @next/next/no-img-element */
+import styles from "./MaintainerCard.module.css"
+
+export interface MaintainerCardProps {
+	maintainerImage: string
+	maintainerName: string
+	maintainerSurname: string
+	maintainerDescription: string
+	maintainerGithubName: string
+	maintainerGithubLink: string
+}
+
+export const MaintainerCard = (props: MaintainerCardProps) => {
+	const {
+		maintainerImage,
+		maintainerName,
+		maintainerSurname,
+		maintainerDescription,
+		maintainerGithubName,
+		maintainerGithubLink,
+	} = props
+
+	return (
+		<section>
+			{maintainerImage ? (
+				<img
+					className={styles.maintainerImage}
+					src={maintainerImage}
+					alt={`${maintainerName} ${maintainerSurname}`}
+				/>
+			) : (
+				<p className={styles.maintainerImage}>{maintainerImage || "image"}</p>
+			)}
+			<section className={styles.maintainerContainer}>
+				<h1 className={styles.maintainerFullName}>
```

Comment:
h1 is not the right element to use here. I'd suggest that it's not a heading at all so a p tag with styling is more than adequate.